api/routes/workflow_refactored.py

```
@@ -0,0 +1,107 @@
+
+# === ADMIN ENDPOINTS ===
+
+@admin_router.post("/youtube-to-tiktok")
+async def admin_create_youtube_to_tiktok_workflow(request: Request) -> Dict[str, Any]:
+    """Create YouTube to TikTok conversion workflow - Admin endpoint (enhanced features)"""
+    try:
+        data = await request.json()
+        return workflow_service.create_youtube_to_tiktok_workflow(data, is_admin=True)
+    except Exception as e:
+        logger.error(f"Admin YouTube to TikTok workflow creation failed: {e}")
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)